angular-webflux/angular.json

```diff
@@ -110,7 +110,7 @@
   "cli": {
     "cache": {
       "enabled": true,
-      "path": "./target/",
+      "path": "./target//angular/",
```

Comment:
@mshima I'm not sure why this changed.

Reply:
It’s a generator-jhipster change.
Angular was cleaning the entire target folder and builds were failing.